src/EventListener/DeserializeListener.php

```diff
@@ -51,6 +51,7 @@ public function onKernelRequest(GetResponseEvent $event)
             || $request->isMethod(Request::METHOD_DELETE)
             || !($attributes = RequestAttributesExtractor::extractAttributes($request))
             || !$attributes['receive']
+            || ($request->isMethod(Request::METHOD_PUT) && $request->getContent() === '')
```

Comment:
Can you store the result of `$request->getContent()` in a variable to avoid 2 (costly) method calls (the next one is line 70)?
Also, please use Yoda styl (`'' === $content`).